ThirdParty/Dependencies

```diff
@@ -1 +1 @@
-Subproject commit 49eb2a3e2190b09433a856508d94b14b649da473
+Subproject commit a62d80be574ef74ee4196e3f7f1a40835634d857
```

Comment:
I'm not sure about updating this to...

https://github.com/MonoGame/MonoGame.Dependencies/commit/a62d80be574ef74ee4196e3f7f1a40835634d857

@cra0zy - Is this safe without other code changes?